specs/pvs/corla.pvs

```diff
@@ -470,6 +495,11 @@ ballot_manifests: THEORY
 BEGIN
   IMPORTING ballots, elections_equipment, information_systems
 
+% @todo kiniry Should this information also include what kind of
+% ballot each ballot is?  I.e., re-marked, phantom, etc.?
+% @todo kiniry These types will be refined when we review the
+% information provided by CDOS on 20 July 2017 about Dominion's file
+% formats.
   ballot_manifest_info: TYPE =
     [scanner_id_number, batch_number, batch_size, storage_container]
   ballot_manifest: TYPE = set[ballot_manifest_info]
```

Comment:
Note that I'm leaving in this comment for the moment since I still do no think we have clarity on the contents of Dominion's ballot manifest.